baseapp/abci_test.go

```diff
@@ -112,6 +112,7 @@ func TestGetBlockRentionHeight(t *testing.T) {
 		})
 
 		t.Run(name, func(t *testing.T) {
+			ps.t = t
```

Comment:
Note that this would prevent these tests to run in parallel (you're sharing the same `paramStore` across multiple tests)

Reply:
we are not doing parallel tests here, but I will update to make it working if this will be enabled here.

Reply:
It should be as easy as create a new instance for each `r.Run()` call. It doesn't seem to difficult.

Reply:
I know, I already did it.